rebar.config

```diff
@@ -1,19 +1,19 @@
 {erl_opts, [debug_info]}.
-{deps, [{riak_core, "3.1.0", {pkg, riak_core_ng}},
-        {lager, "3.5.1"}]}.
+{deps, [{riak_core, {git, "https://github.com/basho/riak_core", {ref, "f15acd3a87150431277eb754792ec24a0dc81d75"}}}]}.
 
-{relx, [{release, {rc_example, "0.1.0"},
-         [rc_example]},
+{shell, [
+  % {config, "config/sys.config"},
+    {apps, [rc_example]}
+]}.
+{relx, [{release, {rc_example, "0.1.0"}, [rc_example]},
         {dev_mode, true},
         {include_erts, false},
-        {extended_start_script, false},
         {sys_config, "conf/sys.config"},
```

Comment:
didn't the duplicated sys_config/vm_args with the overaly templates cause problems after all?

Reply:
Yes, but it's working like this, and the issue has been fixed here: https://github.com/erlang/rebar3/issues/2710